gen3-integration-tests/tests/test_gen3_workflows.py

```diff
@@ -150,117 +201,300 @@ def test_unauthorized_user_cannot_list_tes_tasks(self):
 
     def test_unauthorized_user_cannot_create_tes_tasks(self):
         """
         Ensure that an unauthorized user cannot create TES tasks.
-        Expects: HTTP 401 Unauthorized response.
+        Expects: HTTP 403 Forbidden response.
         """
-        # TODO: Work on 401 for user = None, and 403 for user without `create` policy
-        response = self.gen3_workflow.create_tes_task(
+
+        self.gen3_workflow.create_tes_task(
+            request_body={},
             user=self.invalid_user,
-            expected_status=401,
+            expected_status=403,
         )
 
     def test_happy_path_create_tes_tasks(self):
         """
-        Verify that an authorized user can successfully create a TES task.
-        Expects: HTTP 200 with an 'id' field in the response.
+        Test Case: Happy Path for TES Task Creation
+        - Upload input file to S3
+        - Submit TES task
+        - Verify task creation, listing, retrieval, and completion
+        - Validate outputs and logs
         """
-        response = self.gen3_workflow.create_tes_task(
+        message = "hello beautiful world!"
+        s3_path_prefix = f"{self.s3_storage_config.bucket_name}/tes-test-dir"
+
+        # Step 1: Upload input file to S3
+        self.gen3_workflow.put_bucket_object_with_boto3(
+            content=message,
+            object_path=f"{s3_path_prefix}/input.txt",
+            s3_storage_config=self.s3_storage_config,
             user=self.valid_user,
             expected_status=200,
         )
-        assert (
-            "id" in response
-        ), f"Response should contain a valid 'id' field. But found {response} instead"
 
-    def test_happy_path_list_tes_tasks(self):
-        """
-        Verify that an authorized user can list TES tasks.
-        Expects: HTTP 200 with a non-empty task list containing the newly created task ID.
-        """
-        response = self.gen3_workflow.create_tes_task(
+        # Step 2: Create a TES task
+        tes_task_payload = {
+            "name": "Hello world with Word Count",
+            "description": "Demonstrates the most basic echo task.",
+            "inputs": [
+                {"url": f"s3://{s3_path_prefix}/input.txt", "path": "/data/input.txt"}
+            ],
+            "outputs": [
+                {
+                    "path": "/data/output.txt",
+                    "url": f"s3://{s3_path_prefix}/output.txt",
+                    "type": "FILE",
+                },
+                {
+                    "path": "/data/grep_output.txt",
+                    "url": f"s3://{s3_path_prefix}/grep_output.txt",
+                    "type": "FILE",
+                },
+            ],
+            "executors": [
+                {
+                    "image": "public.ecr.aws/docker/library/alpine:latest",
+                    "command": [
+                        "cat /data/input.txt > /data/output.txt && grep hello /data/input.txt > /data/grep_output.txt && echo Done!"
```

Comment:
could we store "Done!" in a variable so it's easy to understand why we look for "Done!" in the output later?